attractions/_mixins.scss

```diff
@@ -15,3 +18,29 @@
   overflow: visible;
   position: static;
 }
+
+@mixin show-on-focus {
+  @include hide-accessible;
+
+  &:focus {
+    @include show-accessible;
+  }
+}
+
+@mixin slim-scrollbar {
```

Comment:
I kinda tried to do it but then realized that this would be too unfitting and it's not even something they'd commonly use. I exposed them for people who want to mimic some components with their own functionality and still be able to style them as we do. This is for an explorer's eye, they'll figure it out themselves